pkg/controller/kubedirectorcluster/members.go

```diff
@@ -577,6 +651,16 @@ func handleCreatingMembers(
 				}
 			}
 
+			backupErr := handleConfigPackage(reqLogger, cr, m)
+
+			if backupErr != nil {
+				statusErrMsg := fmt.Sprintf(
+					"failed to backup config package after upgrade: %s",
+					backupErr.Error(),
+				)
+				setFinalState(memberConfigError, &statusErrMsg)
+			}
```

Comment:
We would need to `return` here in the error case, correct?

Reply:
Sure